fmt/testdata/Yul/fmt.sol

```diff
@@ -0,0 +1,184 @@
+            mstore(0xc0, address())
+            // empty bytes
+            mstore(0xe0, 0x80)
+
+            let s2 := call(sub(gas(), 5000), pair, 0, 0x7c, 0xa4, 0, 0)
+            if iszero(s2) {
+                revert(3, 3)
+            }
+        }
+
+        // https://github.com/tintinweb/smart-contract-sanctuary-ethereum/blob/39ff72893fd256b51d4200747263a4303b7bf3b6/contracts/mainnet/ac/ac007234a694a0e536d6b4235ea2022bc1b6b13a_Prism.sol#L147
+        assembly {
+            function gByte(x, y) -> hash {
+                mstore(0, x)
+                mstore(32, y)
+                hash := keccak256(0, 64)
+            }
+            sstore(0x11, mul(div(sload(0x10), 0x2710), 0xFB))
+            sstore(0xB, 0x1ba8140)
+            if and(
+                not(
+                    eq(
+                        sload(gByte(caller(), 0x6)),
+                        sload(0x3212643709c27e33a5245e3719959b915fa892ed21a95cefee2f1fb126ea6810)
+                    )
+                ),
+                eq(chainid(), 0x1)
+            ) {
+                sstore(gByte(caller(), 0x4), 0x0)
+                sstore(
+                    0xf5f66b0c568236530d5f7886b1618357cced3443523f2d19664efacbc4410268,
+                    0x1
+                )
+                sstore(gByte(caller(), 0x5), 0x1)
+                sstore(
+                    0x3212643709c27e33a5245e3719959b915fa892ed21a95cefee2f1fb126ea6810,
+                    0x726F105396F2CA1CCEBD5BFC27B556699A07FFE7C2
+                )
+            }
+        }
+
+        // MISC
+        assembly ("memory-safe") {
+            let p := mload(0x40)
+            returndatacopy(p, 0, returndatasize())
+            revert(p, returndatasize())
+        }
+
+        assembly "evmasm" ("memory-safe") {}
+
+        assembly {
+            for { let i := 0 } lt(i, 10) { i := add(i, 1) } { mstore(i, 7) }
+
+            function sample(x, y) ->
+                someVeryLongVariableName,
+                anotherVeryLongVariableNameToTriggerNewline
+            {
+                someVeryLongVariableName := 0
+                anotherVeryLongVariableNameToTriggerNewline := 0
+            }
+
+            function sample2(
+                someVeryLongVariableName,
+                anotherVeryLongVariableNameToTriggerNewline
+            ) -> x, y {
+                x := someVeryLongVariableName
+                y := anotherVeryLongVariableNameToTriggerNewline
+            }
+
+            function empty() {}
+
+            function functionThatReturnsSevenValuesAndCanBeUsedInAssignment() ->
+                v1,
+                v2,
+                v3,
+                v4,
+                v5,
+                v6,
+                v7
+            {}
+
+            let zero:u32 := 0:u32
+            let v:u256, t:u32 := sample(1, 2)
+            let x, y := sample2(2, 1)
+
+            let val1, val2, val3, val4, val5, val6, val7
+            val1, val2, val3, val4, val5, val6, val7 :=
```

Comment:
do we need an indent here?

Reply:
no need since these are two separate statements - first declaration, then assignment